vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -977,6 +978,36 @@ void NativeUIManager::measureInWindow(
   callback(args);
 }
 
+void NativeUIManager::measureLayout(
+    facebook::react::ShadowNode &shadowNode,
+    facebook::react::ShadowNode &ancestorNode,
+    facebook::xplat::module::CxxModule::Callback errorCallback,
+    facebook::xplat::module::CxxModule::Callback callback) {
+  std::vector<folly::dynamic> args;
+  try {
+    const auto &target = static_cast<ShadowNodeBase &>(shadowNode);
+    const auto &ancestor = static_cast<ShadowNodeBase &>(ancestorNode);
+    const auto targetView = target.GetView().as<winrt::FrameworkElement>();
+    const auto ancenstorView = ancestor.GetView().as<winrt::FrameworkElement>();
+
+    const auto ancestorViewtransform = targetView.TransformToVisual(ancenstorView);
+    const auto relativePosition = ancestorViewtransform.TransformPoint({0, 0});
+
+    // x, y
+    args.push_back(relativePosition.X);
+    args.push_back(relativePosition.Y);
+
+    // Size
+    args.push_back(targetView.ActualWidth());
+    args.push_back(targetView.ActualHeight());
```

Comment:
Does width/height need to be transformed into the ancestor coordinate space as well?  I would think yes.  For example, imagine there's a scale transform set on the target element.  It would scale the width/height when transforming up to the ancestor.

Reply:
Updated : )